src/components/Header/Header.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import { NewTodoForm } from '../NewTodoForm';
+
+export const Header: React.FC = () => (
```

Comment:
The component name and file name do not match. Rename the file to 'Header.tsx' or the component to 'Header'